backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -64,7 +64,7 @@ public TestOrderService(
   @AuthorizationConfiguration.RequirePermissionStartTest
   public List<TestOrder> getQueue(String facilityId) {
     Facility fac = _os.getFacilityInCurrentOrg(UUID.fromString(facilityId));
-    return _repo.fetchQueue(fac.getOrganization(), fac);
+    return _repo.fetchQueue(_os.getCurrentOrganization(), fac);
```

Comment:
The old call is secure if you dig down. Just moved the call to getCurrentOrganization up so as to surface it at this level to make it obvious what's going on.

Reply:
I disagree with the change--fetching the org twice just to make a point doesn't seem justified.